<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMacros</class>
 <widget class="QDialog" name="DialogMacros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="tree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>411</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau dossier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouvelle macro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer la macro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Raccourci :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKeys">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot sur la gauche :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLeftWord">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReadOnly">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Verrouill√©</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
